### ISSUE TITLE
KeyError in [CompileList] (Problem with NCS compiler?)

### ISSUE THREAD

**Issue by th3w1zard1** [NickHugi/PyKotor#6]
Had this error just a few times but for multiple mods:
```
[7/2/2023 7:10:11 PM] Using changes.ini path: C:\Users\*****\Documents\k1 mods\KillCzerkaJerk\tslpatchdata\changes.ini

[7/2/2023 7:10:12 PM] Traceback (most recent call last):
  File "pykotorcli.py", line 66, in <module>
  File "pykotor\tslpatcher\config.py", line 118, in install
  File "pykotor\tslpatcher\config.py", line 113, in config
  File "pykotor\tslpatcher\config.py", line 77, in load
  File "pykotor\tslpatcher\reader.py", line 57, in load
  File "pykotor\tslpatcher\reader.py", line 202, in load_nss
  File "configparser.py", line 979, in __getitem__
KeyError: 'kas22_attack.nss'
```

Relevant `changes.ini` example that'll reproduce:
```
; =====================================================[v1.0.5b1]====
; TSLPATCHER - GENERATED MODIFICATIONS FILE (25/03/2021)
; ===================================================================
; This file is automatically generated and as such has no formatting
; to speak of. You can insert blank lines between sections (but NOT
; between keys within a section!) and add comment lines starting
; with semicolon to make it more readable without breaking anything.
; -------------------------------------------------------------------

[Settings]
FileExists=1
WindowCaption=Request: Kill the Czerka jerk on Kashyyyk
ConfirmMessage=N/A
LogLevel=3
InstallerMode=1
BackupFiles=1
PlaintextLog=0
LookupGameFolder=0
LookupGameNumber=2
SaveProcessedScripts=0


[TLKList]


[InstallList]
install_folder0=Override


[2DAList]


[GFFList]


[CompileList]
Replace0=kas22_attack.nss


[SSFList]


; ===================================================================

[install_folder0]
Replace0=kas22_czerkag_01.dlg
Replace1=kas22_czerkag_01.utc
Replace2=kas22_czerkag_02.utc
Replace3=kas22_czerkag_03.utc
Replace4=kas22_czerkag_04.utc
Replace5=kas22_czerka2_01.utc

```

**Comment by th3w1zard1** [comment]
Fixed by #7

**Comment by th3w1zard1** [comment]
Problem somehow reappeared

**Comment by th3w1zard1** [comment]
Fixed in #18
